org.kde.trojita.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.kde.trojita
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: "5.15-21.08"
+command: trojita
+rename-desktop-file: trojita.desktop
+rename-icon: trojita
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --persist=.abook
+
+modules:
+  - name: trojita
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DWITH_TESTS=OFF
+      - -DWITH_QT5=ON
+      - -DWITH_ZLIB=ON
+      - -DWITH_RAGEL=OFF
+    post-install:
+      - rm ${FLATPAK_DEST}/share/appdata/trojita.appdata.xml
+      - install org.kde.trojita.appdata.xml ${FLATPAK_DEST}/share/appdata/org.kde.trojita.appdata.xml
+    sources:
+      - type: git
+        url: https://invent.kde.org/pim/trojita.git
```

Comment:
I think KDE maintainers prefer the use of tarball and json.

Reply:
Yes to both

Reply:
Though the problem with tarball is that last release of trojita was 5 years ago, but anyway, we should only package released things, 5 years old version is too old, you should pressure Jan into doing a release.